### PR TITLE
cluster: display dashboard server URLs

### LINKED ISSUE #2470: Problems that standalone Dashbaord displays in tiup
## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do?

Deploy the dashboard using the Tiup service independently.

2. What did you expect to see?
Display the correct Dashboard URL in Tiup

3. What did you see instead?
The dashboard address displayed by Tiup is still the address of pd
Dashboard has multiple to-display lists，like DashboardURLs
if PD Dashboard was closed，should not be displayed

4. What version of TiUP are you using (`tiup --version`)?
1.16.0 v1.16.0-nightly-11

### PR DESCRIPTION
**Pull request by xhebox**
Signed-off-by: xhe <xw897002528@gmail.com>

### What problem does this PR solve? 

Close #2470

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 
aae6846c-a798-43d1-adb3-eb99b934c8cb

 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

Release notes:

```release-note
NONE
```

### CODE REVIEW

File: pkg/cluster/manager/display.go [hunk 1]
```diff
@@ -801,38 +789,52 @@ func (m *Manager) DisplayDashboardInfo(clusterName string, timeout time.Duration
...
 	}
 
-	u.Path = "/dashboard/"
+	for i, addr := range dashboardAddrs {
+		scheme := "http"
 
-	if tlsCfg != nil {
-		fmt.Println(
-			"Client certificate:",
-			color.CyanString(m.specManager.Path(clusterName, spec.TLSCertKeyDir, spec.PFXClientCert)),
```
Comment (xhebox) [inline pkg/cluster/manager/display.go:825]: I believe PFX format is deprecated, right @kaaaaaaang ?

Comment (kaaaaaaang) [reply pkg/cluster/manager/display.go:825]: I don't know anything about its deprecation.

**Review by kaaaaaaang — COMMENTED** [review]
rest /lgtm

File: pkg/cluster/manager/display.go (hunk 1 above)
Comment (xhebox) [reply pkg/cluster/manager/display.go:825]: Bring back the old code for now.

**Pull request merged**